.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
+node_modules
+packages/*/node_modules
```

Comment:
Can we also remove the source files?

Reply:
yea added to the Dockerfile as rm command.